src/js/starter_app.js

```diff
@@ -0,0 +1,142 @@
+(function(window){
+  window.extractData = function() {
+    var ret = $.Deferred(); 
+  
+    function onError() {
+      console.log('Loading error', arguments);
+      ret.reject();
+    }     
+
+    function onReady(smart)  {
+      if (smart.hasOwnProperty('patient')) { 
+        var patient = smart.patient;
+        var pt = patient.read();
+        var obv = smart.patient.api.fetchAll({
+                      type: 'Observation', 
+                      query: {
+                        code: {
+                          $or: ['http://loinc.org|8302-2', 'http://loinc.org|8462-4',
+                                'http://loinc.org|8480-6', 'http://loinc.org|2085-9',
+                                'http://loinc.org|2089-1']
```

Comment:
You might want to request the grouper loinc for blood pressure here since our fhir service will respond to that. 
LOINC :  55284-4
Ref : https://github.cerner.com/Igneous/igneous-smart_server/blob/master/public/cardiac-risk-app-2.0/src/js/cardiac_risk.js#L53
Ref Usage : https://github.cerner.com/Igneous/igneous-smart_server/blob/master/public/cardiac-risk-app-2.0/src/js/cardiac_risk.js#L148

If we dont include this grouper its possible that we might not get any observations back for sysBP and diasBP.